### ISSUE TITLE
is it possible to use cuda

### ISSUE THREAD

**Issue by twmht** [staghado/vit.cpp#9]
Hi,

is it possible to use cuda for inference?

**Comment by staghado** [comment]
It is possible : refer to the [GGML](https://github.com/ggerganov/ggml?tab=readme-ov-file#using-cublas) library.

**Comment by twmht** [comment]
yeah i know ggml supports cuda, what i mean is do you have any plan to add options for cuda?

**Comment by staghado** [comment]
I can add it if that's something you need. One issue is that i do not have a local GPU so it will be hard to test. would you like to test? you can also open a PR if you like!
